release_dashboard/lib/services/local_conductor.dart

```diff
@@ -28,19 +28,24 @@ class LocalConductorService extends ConductorService {
     stdin: io.stdin,
   );
 
+  Directory? _engineCheckoutDirectory;
+  Directory? _frameworkCheckoutDirectory;
+
   @override
   Directory get rootDirectory => fs.directory(platform.environment['HOME']);
   File get stateFile => fs.file(defaultStateFilePath(platform));
 
-  static const String frameworkUpstream = 'https://github.com/flutter/flutter';
-  static const String engineUpstream = 'https://github.com/flutter/engine';
+  @override
+  Directory get engineCheckoutDirectory => _engineCheckoutDirectory ?? rootDirectory;
```

Comment:
Shouldn't this throw if `_engineCheckoutDirectory` is null?

Reply:
I changed it so it would throw a `LateInitializationError`.